src/js/with-react.js

```diff
@@ -70,6 +70,12 @@ const withReact = (WebComponent, { pure = true, passive = false, eventNamespace
     constructor(props) {
```

Comment:
May we could also remove all lifecycle hooks for tests or write a separate `withReactForTests` HOC 🤔